docs/commands/oras_cp.mdx

```diff
@@ -46,8 +46,7 @@ oras cp -r localhost:5000/net-monitor:v1 localhost:6000/net-monitor-copy:v1
 Copy an artifact and referrers using specific methods for the Referrers API:
 
 ```bash
-oras cp -r --from-distribution-spec v1.1-referrers-api --to-distribution-spec v1.1-referrers-tag \
-    localhost:5000/net-monitor:v1 localhost:6000/net-monitor-copy:v1
+oras cp -r --from-distribution-spec v1.1-referrers-api --to-distribution-spec v1.1-referrers-tag     localhost:5000/net-monitor:v1 localhost:6000/net-monitor-copy:v1
```

Comment:
Was this intentional?

Reply:
Created https://github.com/oras-project/oras-www/issues/322 to track the change your comment was about. The shell script currently joins lines ending with `\`